Server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const port = 3000;
```

Comment:
It's a good practice to externalize configurations like the server port. Consider using environment variables for this. [medium]